go/internal/feast/featurestore.go

```diff
@@ -654,14 +654,17 @@ func (fs *FeatureStore) listFeatureViews(hideDummyEntity bool) ([]*FeatureView,
 	if err != nil {
 		return featureViews, err
 	}
-	for _, featureView := range featureViews {
-		if _, ok := featureView.entities[DUMMY_ENTITY_NAME]; ok && hideDummyEntity {
-			featureView.entities = make(map[string]struct{})
-		}
-	}
 	return featureViews, nil
 }
 
+func (fs *FeatureStore) listRequestFeatureViews() ([]*RequestFeatureView, error) {
+	requestFeatureViews, err := fs.registry.listRequestFeatureViews(fs.config.Project)
+	if err != nil {
+		return requestFeatureViews, err
+	}
+	return requestFeatureViews, nil
+}
```

Comment:
can't this just be 
```
	return fs.registry.listRequestFeatureViews(fs.config.Project)
```

Reply:
Ah originally I added the hideDummyEntity but that's irrelevant here.